mobile/evidence/ios/Leaf/Sources/Leaf/Theme/LeafTheme.swift

```diff
@@ -2,46 +2,28 @@ import SwiftUI
 
 public struct LeafTheme {
     public let color: Color
-    public let font: Font
     public let size: Size
 }
 
 extension LeafTheme {
     public struct Color {
-        public let brand: BrandColor
-        public let content: ContentColor
-        public let tag: TagColor
+        public let system: SystemColors
+        public let custom: CustomThemeColors
     }
-    
-    public struct BrandColor {
-        public let primary: SwiftUI.Color
-        public let secondary: SwiftUI.Color
-        public let tertiary: SwiftUI.Color
-    }
-    
-    public struct ContentColor {
+
+    public struct SystemColors {
         public let primary: SwiftUI.Color
         public let secondary: SwiftUI.Color
-        public let tertiary: SwiftUI.Color
-        public let quaternary: SwiftUI.Color
+        public let link: SwiftUI.Color
+        public let successGreen: SwiftUI.Color
+        public let successBlue: SwiftUI.Color
```

Comment:
I would avoid using specific color names in the color tokens. Maybe `success` and `successLight` would be a little bit better.